examples/auth/app/users/mutations/updateUser.ts

```diff
@@ -0,0 +1,17 @@
+import {protect} from "blitz"
+import db from "db"
+import * as z from "zod"
+
+export default protect(
+  {
+    schema: z.object({
```

Comment:
should the example really put the schemas inline like this? I like extracting and exporting it so I can re-use in my forms for validation.